input/tail.md

```diff
@@ -402,6 +412,80 @@ The `@log_level` option allows the user to set different levels of logging for e
 
 Refer to the [Logging](../deployment/logging.md) for more details.
 
+
+### `<group>` Section \(highly recommended\)
```

Comment:
```suggestion
### `<group>` Section
```

I think this function is rather optional.
(Use when we want more detailed control than `read_bytes_limit_per_second`)

Reply:
Removed in c0a9c1e35276d9e33ec8280b6a33f6aa9c46c253